docs/source/java/overview.rst

```diff
@@ -44,6 +44,9 @@ but some modules are JNI bindings to the C++ library.
    * - arrow-memory-netty
      - Memory management implementation based on Netty.
      - Native
+   * - arrow-memory-ffm
```

Comment:
Since the Java API itself is just called "foreign" perhaps just go with their naming and arrow-memory-foreign rather than the acronym?

Reply:
SGTM! Good suggestion, will update.